recipes/carve/build.sh

```diff
@@ -0,0 +1,8 @@
+if [ "$(uname)" == "Darwin" ]; then
+    cmake  ${CMAKE_ARGS} -DCMAKE-INSTALL-PREFIX=${PREFIX} .      
+elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+    export CXXFLAGS="$CXXFLAGS -std=c++14"
```

Comment:
Do you really need C++14? We normally compile with C++17. Please add a comment.

Reply:
Unfortunately yes. Noticing that osx compiles with C++ 14 was the key to understanding why the OSX build was successful whilst the Linux build wasn't. I will add the comment on the file.

Reply:
Apologies, I am late for the party. Just a quick note: I prefer using `-DCMAKE_CXX_STANDARD=14` instead of passing `-std=c++14` manually to `CXXFLAGS` when using `cmake`